addons/message_update_v2.23/manifest.json

```diff
@@ -4,7 +4,8 @@
   "name": "Update to Mozilla VPN 2.23",
   "type": "message",
   "conditions": {
-    "max_client_version": "2.22.9"
+    "max_client_version": "2.22.9",
+    "javascript": "osCheck.js"
   },
   "javascript": {
     "enable": "enable.js"
```

Comment:
Nice! Any reason to not just do this in enable.js where the other show/hide decision making happens?

Reply:
`enable` is run when the addon is enabled. We use it to modify the data in the addon based on conditions. 

There is an addon condition type (conditions are used to decide whether to even show the addon) of `javascript`, which can enable arbitrary javascript as part of a test as to whether to show the addon. It seems more appropriate to use a `condition` for this, and I'm not sure it's even possible to use an `enable` js snippet to halt usage of the addon. If you know of a way, I'm happy to explore.

Reply:
Discussed in person, we're all comfortable w/ this. Going to merge.